packages/design-system-dashboard-cli/src/find-ds-components.js

```diff
@@ -163,18 +166,30 @@ function findComponents(searchStrings) {
   );
 
   const wcTagRegex = /<(va-[^\s>]+)/gms;
-  const vwWebComponents = search(vwModules, wcTagRegex);
-  const contentBuildWC = search(contentTemplates, wcTagRegex);
+  const wcUswds3Regex = /<(Va[^\s]+|va-[^\s]+)(\s|\n)[^>]*?uswds/gms;
```

Comment:
This regex matches either react binding usage by looking for `<Va` or regular web component usage by looking for `<va`. Then, it looks for the presence of `uswds` either on the same line or accounting for multiline prop declarations.

Reply:
I'm a fan of https://regex101.com/. I like how the site explains each segment of a regex and has a quick reference window for tokens.